pkg/sharding/listwatch_test.go

```diff
@@ -25,7 +25,7 @@ import (
 )
 
 func TestSharding(t *testing.T) {
-	cm := &v1.ConfigMap{
+	cm := &v1.Pod{
 		ObjectMeta: metav1.ObjectMeta{
 			Name:      "configmap1",
```

Comment:
Seems like an odd name for a Pod :wink:

Reply:
heh yeah I just changed this one to make the Test still work ;) I can change that though, good catch. :D